src/views/LegalNoticeView.vue

```diff
@@ -16,9 +16,9 @@ const navigateToHome = () => {
     </div>
     <div class="col-6">
       <Card>
-        <template #title> Impressum </template>
+        <template #title> {{ $t('legalNotice.title') }}</template>
         <template #content>
-          <p>Angaben gemäß § 5 Telemediengesetz (TMG):</p>
+          <p>{{ $t('legalNotice.definition') }}</p>
           <p>
             Prof. Dr.-Ing. Alexander Stanik<br />
```

Comment:
It’s not necessary to translate the name here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.